code/modules/organs/robolimbs.dm

```diff
@@ -475,7 +475,12 @@ var/const/cyberbeast_monitor_styles= "blank=cyber_blank;\
 	species_alternates = list(SPECIES_TAJ = "Zeng-Hu - Tajaran")
 	unavailable_to_build = TRUE
 	skin_tone = TRUE
-
+	
+/datum/robolimb/spectre
+	company = "Hammond Tech - RACS Spectre "
+	desc = "A simple robotic limb design used for the Hammond Tech RASC Spectre. A lightweight robotic chassis ideal for exploration and security duties."
+	icon = 'icons/mob/cyberlimbs/_fluff_vr/spectre.dmi'
```

Comment:
Repath this to something like `'icons/mob/cyberlimbs/hammond_tech/spectre.dmi'`, `/_fluff_vr/` is legacy. 
We currently path based on `/company/model.dmi` for maintainability. But yeah obviously `hammond_tech` in my suggestion is placeholder for whatever the loremaints decide on.